django/db/migrations/operations/base.py

```diff
@@ -111,6 +111,15 @@ def allow_migrate_model(self, connection_alias, model):
 
         return router.allow_migrate_model(connection_alias, model)
 
+    def reduce(self, operation, in_between, app_label=None):
+        """
+        Either returns a list of zero, one or two operations, meaning the
```

Comment:
Ah, yeah, `True` is not explicitly checked and just a placeholder which is neither a list nor `None`. So `False` is meaningless. However, since `True` is mentioned explicitly I'd probably change `None` to `False`.